core/dynamic-tags/dynamic-css.php

```diff
@@ -13,6 +13,11 @@
 class Dynamic_CSS extends Post {
 
 	protected $post_id_for_data;
+
+	protected function is_global_parsing_supported() {
```

Comment:
Consider extend a new Base_Post class that doesn't set `is_global_parsing_supported=true`.

Reply:
But now it's:
 `Base=false -> Post=true -> Dynamic=false`.
Instead it can be: 
`Base=false -> Base_Post=inherit -> Dynamic=inherit`